website/html/views.py

```diff
@@ -1107,7 +1108,17 @@ def _build_installer(username, platform):
     return False, error_response
 
   try:
-    xmlrpc_proxy = xmlrpclib.ServerProxy(settings.SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC)
+    # Per default this setting is True in DEBUG mode.
+    # Passing an unverified ssl context allows the Custom Installer Builder
+    # to use self-signed server certificates.
+    # c.f. https://github.com/SeattleTestbed/custominstallerbuilder/issues/16
+    if settings.UNVERIFIED_SSL_CONTEXT:
+      xmlrpc_proxy = xmlrpclib.ServerProxy(
+        settings.SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC, 
+        context=ssl._create_unverified_context())

```

Comment:
AFAIK we agreed that we don't care for Python <2.7 to run clearinghouse (c.f. discussion in SeattleTestbed/custominstallerbuilder#16).

As for the disclaimer, our main instruction to run the clearinghouse already [suggests to use Python 2.7](https://github.com/SeattleTestbed/docs/blob/master/Operating/Clearinghouse/Installation.md)